languages/es_MX.php

```diff
@@ -0,0 +1,71 @@
+$lang['regester_success_emailmessage_suppressed'] = "La cuenta se ha creado.";
+$lang['email_taken'] = "El correo electrónico está actualmente en uso.";
+
+$lang['resetkey_invalid'] = "La clave de reinicio es inválida.";
+$lang['resetkey_incorrect'] = "La clave de reinicio es incorrecta.";
+$lang['resetkey_expired'] = "La clave de reinicio ha expirado.";
+$lang['password_reset'] = "La contraseña ha sido reiniciada con éxito.";
+
+$lang['activationkey_invalid'] = "La clave de activación es inválida.";
+$lang['activationkey_incorrect'] = "La clave de activación es incorrecta.";
+$lang['activationkey_expired'] = "La clave de activación ha expirado.";
+
+$lang['reset_requested'] = "La petición para reiniciar la contraseña ha sido enviado al correo electrónico.";
+$lang['reset_requested_emailmessage_suppressed'] = "La petición para reiniciar la contraseña ha sido creada.";
+$lang['reset_exests'] = "Ya existe una petición para reiniciar la contraseña.";
+
+$lang['already_activated'] = "La cuenta ya ha sido activada.";
+$lang['activation_sent'] = "El correo electrónico de activación ha sido enviado.";
+$lang['activation_exests'] = "El correo electrónico de activación ya ha sido enviado.";
+
+$lang['email_activation_subject'] = '%s - Activar cuenta';
+$lang['email_activation_body'] = 'Hola,<br/><br/>Para inciar sesión en tu cuenta de usuario primero tienes que activar tu cuenta dando click en el siguiente enlace: <strong><a href="%1$s/%2$s">%1$s/%2$s</a></strong><br/><br/> Después, necesitas usar la siguiente clave de activación: <strong>%3$s</strong><br/><br/> Si no te registraste en %1$s recientemente entonces este mensaje fue enviado por error, por favor ignóralo.';
+$lang['email_activation_altbody'] = 'Hola, ' . "\n\n" . 'Para inciar sesión en tu cuenta de usuario primero tienes que activar tu cuenta dando click en el siguiente enlace :' . "\n" . '%1$s/%2$s' . "\n\n" . 'Después, necesitas usar la siguiente clave de activación: %3$s' . "\n\n" . 'Si no te registraste en %1$s recientemente entonces este mensaje fue enviado por error, por favor ignóralo';
+
+$lang['email_reset_subject'] = '%s - Petición para reinicia contraseña';
+$lang['email_reset_body'] = 'Hola,<br/><br/>Para reiniciar tu contraseña por favor da click en el siguiente enlace:<br/><br/><strong><a href="%1$s/%2$s">%1$s/%2$s</a></strong><br/><br/>Después, necesitas usar la siguiente clave de reinicio: <strong>%3$s</strong><br/><br/>Si no solicitaste reiniciar tu contraseña en %1$s por favor ignora este mensaje.';
+$lang['email_reset_altbody'] = 'Hola, ' . "\n\n" . 'Para reiniciar tu contraseña por favor da click en el siguiente enlace:' . "\n" . '%1$s/%2$s' . "\n\n" . 'Después, necesitas usar la siguiente clave de reinicio: %3$s' . "\n\n" . 'Si no solicitaste reiniciar tu contraseña en %1$s por favor ignora este mensaje.';
+
+$lang['account_deleted'] = "La cuenta se ha borrado con éxito.";
+$lang['function_desabled'] = "Esta función ha sido desactivada.";
```

Comment:
I think this is your problem, take a look at https://github.com/PHPAuth/PHPAuth/blob/master/languages/en_GB.php#L71

It's "disabled" not "desabled"